config/sources/families/include/meson64_common.inc

```diff
@@ -22,9 +22,16 @@ GOVERNOR=${GOVERNOR:-ondemand}
 
 case $BRANCH in
 
+	legacy)
+		KERNELSOURCE='https://github.com/hardkernel/linux'
+		#KERNELBRANCH='branch:odroidg12-4.9.y'
+		KERNELBRANCH='tag:4.9.277-122'
+		KERNELDIR='linux-odroid'
+		;;
+
 	current)
-		KERNELBRANCH='branch:linux-5.19.y'
-		KERNELPATCHDIR='meson64-edge'
+		KERNELBRANCH='branch:linux-5.10.y'
```

Comment:
This part was also changed last week. Needs adjustment. CURRENT and EDGE are the same until we add kernel 6.x